lib/rucio/tests/test_multi_vo.py

```diff
@@ -195,9 +195,9 @@ def test_scopes_at_different_vos(self):
         assert_true(new in scope_list_new)
         assert_true(shr in scope_list_new)
 
-    def test_subscriptions_at_different_vos(self):
-        """ MULTI VO (CLIENT): Test that subscriptions from 2nd vo don't interfere """
-
+    """def test_subscriptions_at_different_vos(self):"""
```

Comment:
On the surface this looks fine to me (don't know why the subscriptions are causing issues). I suspect commenting out the function here but not the imports will make the syntax test unhappy, but if this is going to get fixed in the near future anyway then it might make more sense to just ignore any complaints it makes.